### DISCUSSION TITLE
CSV with cointracker

### DISCUSSION THREAD

**Discussion by RyanFact** [davetapley/helium-tax#75] [Q&A]
How do i make this CSV file work with cointracker.

**Comment by davetapley** [comment]
@RyanFact I'm not familiar with cointracker but [this page](https://community.cointracker.io/t/convert-any-csv-into-the-cointracker-csv-format/553) says:

> Refer to the [CoinTracker CSV format](https://www.cointracker.io/add_wallet?t=csv&csv=true#csv-instructions-modal)

However I can't get to that link without signing in.
If you can get to it can you screenshot / paste what it says here, thanks.
